controllers/controller.js

```diff
@@ -36,4 +36,12 @@ exports.getUsers = (req, res, next) => {
       res.status(200).send({ users });
     })
     .catch((err) => next(err));
-};
\ No newline at end of file
+};
+
+exports.getCommentsById = (req, res, next) => {
+  selectCommentsById(article_id)
+  .then((article) => {
+    res.status(200).send({ article });
```

Comment:
Should be sending `comments` instead of article, because the data we are sending is an array of comments from a specific article.

Reply:
Fixed, thanks!